topo_processor/util/tests/transfer_collection_test.py

```diff
@@ -76,7 +79,7 @@ def test_generate_summaries(setup: str) -> None:
     metadata_loader_imagery_historic.add_spatial_extent(item_1, asset_metadata=test_geom)
     item_1.datetime = test_datetime
     item_1.properties = {
-        "mission": "SURVEY_1",
+        "mission": "AUCKLAND 1",
```

Comment:
I just clarified with @dwsilk that `mission` should remain as `survey` as it was previously, not the new `title`.